dexter/src/test/java/com/karumi/dexter/DexterInstanceTest.java

```diff
@@ -70,20 +70,21 @@
   }
```

Comment:
We should cover the new scenario with tests and update the project documentation. Right now it's not obvious this ``shouldForcePremission`` param should be used if you are developing a system app.